### ISSUE TITLE
[7.4.7] Unexpected delay on hide screen

### ISSUE THREAD

**Issue by rex-nihilo** [renpy/renpy#2943]
Here's a simplified example of the issue:

```
screen screen_test(pos_x, pos_y):

    frame:
        at screen_test_at(pos_x, pos_y)
        vbox:
            text "test"
            textbutton"close":
                action Return()

transform screen_test_at(pos_x, pos_y):
    xpos 0
    ypos 0
    on start:
        linear 1.0 xpos pos_x ypos pos_y
    on hide:
        linear 1.0 xpos 0 ypos 0
        linear 0.2 alpha 0
```
...
```
label bug_test:
    call screen screen_test(pos_x=400, pos_y=200)
```

With Renpy 7.4.6 and before, it's ok,
with Renpy 7.4.7 there is a delay before hide action (some seconds)

The problem occurs only when there are parameters with the screen or/and the transform.

**Comment by via-my** [comment]
Dealing with the same problem. The delay increases with the time the screen is opened.

**Comment by renpytom** [comment]
Fixed by d80f614620cf1c549013bfdc7cb35d8df3e2e070 .
